samcli/commands/local/cli_common/options.py

```diff
@@ -153,13 +153,25 @@ def warm_containers_common_options(f):
     warm_containers_options = [
         click.option(
             "--warm-containers",
-            help="Optional. Specifies how AWS SAM CLI manages containers for each function.",
+            help="""
+            \b
```

Comment:
A dummy question, what is the `\b` here? I didn't see it in the other help texts.

Reply:
\b is for click formatting to prevent rewrapping, as I need to have new lines in the help text.
without \b, the help text will look like this:
```
--warm-containers [EAGER|LAZY]    Optional. Specifies how AWS SAM CLI manages
                                  containers for each function. Two modes are
                                  available: EAGER: Containers for all
                                  functions are  loaded at startup and persist
                                  between  invocations. LAZY:  Containers are
                                  only loaded when each  function is first
                                  invoked. Those containers  persist for
                                  additional invocations.
```

but with \b, it will look like this
```
--warm-containers [EAGER|LAZY]    Optional. Specifies how AWS SAM CLI manages 
                                  containers for each function.
                                  Two modes are available:
                                  EAGER: Containers for all functions are 
                                  loaded at startup and persist between 
                                  invocations.
                                  LAZY:  Containers are only loaded when each 
                                  function is first invoked. Those containers 
                                  persist for additional invocations.
```

Reply:
Good to know, this is really helpful! Thanks!

Reply:
Good to see the new names! Much better! 👍